src/pkg/packager/lint/lint.go

```diff
@@ -25,120 +22,109 @@ import (
 	"github.com/xeipuuv/gojsonschema"
 )
 
-// ZarfSchema is exported so main.go can embed the schema file
-var ZarfSchema embed.FS
-
-func getSchemaFile() ([]byte, error) {
-	return ZarfSchema.ReadFile("zarf.schema.json")
+// FileLoader is an interface for reading files, it decouples the lint package from the go embed package which enables testing
+type FileLoader interface {
+	ReadFile(path string) ([]byte, error)
```

Comment:
This function is part of the fs.ReadFileFS interface.

https://pkg.go.dev/io/fs#ReadFileFS

Reply:
Thanks, I should have looked up and seen if there were any interfaces that met that.